dashboard/app/models/contact_rollups_pardot_memory.rb

```diff
@@ -10,13 +10,15 @@
 #  data_synced_at       :datetime
 #  data_rejected_at     :datetime
 #  data_rejected_reason :string(255)
+#  delete_from_pardot   :boolean
```

Comment:
Could this be a datetime value? Something along the line of `marked_for_deletion_at`.
It contains more information than a boolean and can helps with auditing/investigating issues.

Reply:
mmm good idea. I'll change it.